assets/src/scripts/routes/common.js

```diff
@@ -108,14 +108,52 @@ export default {
 				// Assign the button
 				let btn = heading.querySelector( 'button' );
 
-				btn.onclick = () => {
+				let listItems = content.children;
+
+				// Handle list items and events
+				Array.prototype.forEach.call( listItems, item => {
+					// Click the anchor tag inside the list item if the list item is clicked.
+					item.onclick = () => {
+						item.firstElementChild.click();
+					};
+
+					// Click the anchor tag inside the list item if the list item is active and 'Enter' is pressed.
+					item.onkeydown = e => {
+						if ( e.which === 13 ) {
+							item.firstElementChild.click();
+						}
+					};
+
+					// Collapse the content menu if user tabs out.
+					item.onblur = e => {
+						if ( item === listItems[listItems.length - 1] && e.relatedTarget.nodeName !== 'LI' ) {
```

Comment:
Based on previous suggested change, this could become the following:

```javascript
				// Handle link blur events
			
				Array.prototype.forEach.call( links, link => {
			
					// Collapse the content menu if user tabs out.
			
					item.onblur = e => {
			
						if ( item === links[links.length - 1] && e.relatedTarget.parentNode.nodeName !== 'LI' ) {
```

We can use the link's native events, and only need to provide a handler for `blur`.